commit_parser.go

```diff
@@ -81,6 +83,8 @@ func newCommitParser(logger *Logger, client gitcmd.Client, jiraClient JiraClient
 		reIssue:                regexp.MustCompile("(?:" + joinedIssuePrefix + ")(\\d+)"),
 		reNotes:                regexp.MustCompile("^(?i)\\s*(" + joinedNoteKeywords + ")[:\\s]+(.*)"),
 		reMention:              regexp.MustCompile(`@([\w-]+)`),
+		reSignOff:              regexp.MustCompile(`Signed-off-by:\s+([\p{L}\s]+)\s+<([\w.@]+)>`),
+		reCoAuthor:             regexp.MustCompile(`Co-authored-by:\s+([\p{L}\s]+)\s+<([\w.@]+)>`),
```

Comment:
These regex ares great and most probably will work 99% of the times, although having `+` in email address is very common, but dependabot has other stories!

```
Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>
```

To be on the safe side I think `\s+([\p{L}\s\-\[\]]+)\s+<([\w+\-\[\].@]+)>` should do the trick for it.